.travis.settings.xml

```diff
@@ -0,0 +1,10 @@
+<settings>
+  <servers>
+    <server>
+      <id>ossrh</id>
+      <username>${env.OSSRH_USERNAME}</username>
+      <password>${env.OSSRH_PASSWORD}</password>
+    </server>
+  </servers>
+  </profiles>

```

Comment:
What about this tag? Is this supposed to be empty (because it isn't in recheck)?

Reply:
Which tag?